scripts/convert_bias_scan_to_calibration.py

```diff
@@ -5,6 +5,7 @@
 
 #Loads mattak, make sure env variable LD_LIBRARY_PATH includes path to your mattak install
 ROOT.gSystem.Load('libmattak.so')
+ROOT.gSystem.Load('librno-g.so')
```

Comment:
I don't think this is actually necessary, but if it fails, then the conversion will fail anyway, so I think it's fine...